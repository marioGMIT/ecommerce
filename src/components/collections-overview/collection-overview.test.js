import React from 'react';
import { shallow } from 'enzyme';

import { Provider } from "react-redux";
import configureMockStore from "redux-mock-store";

import CollectionOverview from './collections-overview.component'

describe('CollectionOverview component', () => {
    let wrapper;

    const mockStore = configureMockStore();
    const store = mockStore({});

    beforeEach(() => {

        wrapper =shallow(
            <Provider store={store}>
                <CollectionOverview collections={[]}/>
            </Provider>
        );
    });

    it('should render CollectionOverview component', () => {
        expect(wrapper).toMatchSnapshot();
    });

});