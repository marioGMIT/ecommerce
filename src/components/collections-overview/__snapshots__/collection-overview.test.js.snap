// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollectionOverview component should render CollectionOverview component 1`] = `ShallowWrapper {}`;
